### DISCUSSION TITLE
Is there currently a way to scan recent comments from a youtube channel for which I am a moderator?

### DISCUSSION THREAD

**Discussion by LaloCo** [ThioJoe/YT-Spammer-Purge#1037] [Q&A]
I think currently scanning recent comments only works for my channel (the one I used to login). I think it would be a good idea to be able to do this for other channels (at least as moderator) just like in recent videos.

I followed Thio's recommendation to set an additional channel as moderator so that the program doesn't have access to the main channel's credentials, just as an additional security layer, but this prevents me from scanning recent comments and limits me to scan by recent videos.

**Comment by Michael-Lemons** [comment]
I scan channels where I am moderator all of the time. All that you can do is report spam and put it on hold which effectively hides the comments. You don't even need to be a moderator to scan channels.

**Reply by LaloCo** [reply]
Can you do this with option 3 "Scan recent comments across your Entire Channel"? I believe that only works for the channel with which you logged in, not any channel you are a moderator to.
Because that is what I am referring to, I would like to scan say the last 10,000 comments, instead of the last 100 videos for instance.

**Reply by Michael-Lemons** [reply]
Option 3 does not ask you for the channel ID. You could use option 2 and put in a very large number, though. I have never used option 3.
